src/ert/run_models/ensemble_experiment.py

```diff
@@ -31,47 +33,30 @@ class EnsembleExperiment(BaseRunModel):
     will always sample parameters.<br>
     """
 
-    def __init__(
-        self,
-        ensemble_name: str,
-        experiment_name: str,
-        active_realizations: list[bool],
-        minimum_required_realizations: int,
-        random_seed: int,
-        config: ErtConfig,
-        storage: Storage,
-        queue_config: QueueConfig,
-        status_queue: SimpleQueue[StatusEvents],
-    ):
-        self.ensemble_name = ensemble_name
-        self.experiment_name = experiment_name
-        self.experiment: Experiment | None = None
-        self.ensemble: Ensemble | None = None
-
-        self._design_matrix = config.analysis_config.design_matrix
-        self._observations = config.observations
-        self._parameter_configuration = config.ensemble_config.parameter_configuration
-        self._response_configuration = config.ensemble_config.response_configuration
-        self._templates = config.ert_templates
-
-        super().__init__(
-            storage,
-            config.runpath_file,
-            Path(config.user_config_file),
-            config.env_vars,
-            config.env_pr_fm_step,
-            config.runpath_config,
-            queue_config,
-            config.forward_model_steps,
-            status_queue,
-            config.substitutions,
-            config.hooked_workflows,
-            total_iterations=1,
-            log_path=config.analysis_config.log_path,
-            active_realizations=active_realizations,
-            random_seed=random_seed,
-            minimum_required_realizations=minimum_required_realizations,
-        )
+    ensemble_name: str
+    experiment_name: str
+    design_matrix: DesignMatrix | None
+    parameter_configuration: list[ParameterConfig]
+    response_configuration: list[ResponseConfig]
+    ert_templates: list[tuple[str, str]]
+    _observations: dict[str, pl.DataFrame] = PrivateAttr()
+    _experiment_id: UUID | None = PrivateAttr(None)
+    _ensemble_id: UUID | None = PrivateAttr(None)
+
+    def __init__(self, **data: Any) -> None:
+        observations = data.pop("observations", None)
+        super().__init__(**data)
+        self._observations = observations
```

Comment:
Create follow up issue for making these serializable? Could probably just be json, not _that_ large.